page-archives.php

```diff
@@ -362,20 +362,30 @@ function insertSponsors($bcbname)
                     <div id="archives_page_title"><?php the_title();?></div>
                     <div id="archives_page_bcb_selector_buttons">
                         <ul id="archives_page_bcb_selector">
-                            <li class="archive_selected"><a class="archives_current" data-contentdiv="archive_1691">BCB Spring 2018</a></li>
-                            <li><a data-contentdiv="archive_1556">BCB Spring 2016</a></li>
-                            <li><a data-contentdiv="archive_1458">BCB Monsoon 2015</a></li>
-                            <li><a data-contentdiv="archive_1366">BCBX AntHill</a></li>
-                            <li><a data-contentdiv="archive_1224">BCB17</a></li>
-                            <li><a data-contentdiv="archive_1057">BCB16</a></li>
-                            <li><a data-contentdiv="archive_931">BCB15</a></li>
-                            <li><a data-contentdiv="archive_785">BCB14</a></li>
-                            <li><a data-contentdiv="archive_636">BCB13</a></li>
-                            <li><a data-contentdiv="archive_479">BCB12</a></li>
-                            <li><a data-contentdiv="archive_399">BCB11</a></li>
-                            <li><a data-contentdiv="archive_324">BCB10</a></li>
-                            <li><a data-contentdiv="archive_220">BCB9</a></li>
-                            <li><a data-contentdiv="archive_3">BCB8</a></li>
+                            <?php
+                            $first = true;
+                            $archive_term_id = get_cat_ID('archives');
+                            $taxonomy_name = 'category';
+                            $termchildren = get_term_children( $archive_term_id, $taxonomy_name );
```

Comment:
Does it have any contract on the sort order.  the manual order went reverse chronologically to surfact last BCB's archives first.